docs/modules/deployment/nav.adoc

```diff
@@ -2,3 +2,4 @@
 - xref:k8s-service.adoc[Kubernetes Service]
 - xref:k8s-sidecar.adoc[Kubernetes Sidecar]
 - xref:systemd.adoc[Systemd Service]
+- xref:aws-lambda.adoc[AWS Lambda]
```

Comment:
```suggestion
- xref:serverless-faas.adoc[Serverless/FaaS environments]
```

Otherwise it looks like we are endorsing a very specific commercial product as a deployment pattern.